xonsh/xontribs_meta.py

```diff
@@ -675,7 +675,7 @@ def define_xontribs():
         ),
         "z": Xontrib(
             url="https://github.com/AstraLuma/xontrib-z",
-            description="Tracks your most used directories, based on 'frecency'.",
+            description="Tracks your most used directories, based on 'frequency'.",
```

Comment:
```suggestion
            description="Tracks your most used directories, based on 'frecency'.",
```
Looks like a typo but it isn't -- it's a portmanteau of "frequency" and "recency"